polytope/polytope.py

```diff
@@ -290,6 +290,32 @@ def intersect(self, other, abs_tol=ABS_TOL):
 
         return reduce(Polytope(iA, ib), abs_tol=abs_tol)
 
+    def translation(self, d):
+        """Translate the Polytope by a given vector.
+
+        Consult L{polytope.polytope._translate} for implementation details.
+
+        @type d: 1d array
+        @param d: The translation vector.
+
+        @rtype: L{Polytope}
+        @return: A translated copy of the Polytope.
+        """
+        newpoly = self.copy()
+        _translate(newpoly, d)
+        return newpoly
+
+    def rotation(self, u, v, theta=None):
+        """Rotate the Polytope. Only simple rotations are implemented at this
```

Comment:
I think that currently, "simple rotations" means "easily implemented rotations" or "rotations that can be achieved in one step".

I've read that for `N` dimensions, the path between any two orientations will require at least `1` and at most `floor(N/2)` rotations, so for `N > 3`, there is now a multi-step transformation required for arbitrary reorientation. I'm not sure yet **why** the rotation cannot be achieved in one step; since, by definition, all the motion can be projected into a plane. I'll have to do more reading when I implement the missing rotation method.